### ISSUE TITLE
v3.2.1: password_expiration accidentally backported?

### ISSUE THREAD

**Issue by killerbees19** [postfixadmin/postfixadmin#235]
Seems like 9d31639eeff5161147a781d303fa1c353c0afa92 backported parts of the new `password_expiration` feature. I get this error multiple times at **list-virtual.php**:

```
$CONF['password_expiration'] has an invalid value, should be 'YES' or 'NO'
```

But this configuration directive isn't available at v3.2. Tested with v3.2.1-1 from Debian. I'll open a Debian bug too when it's fixed, so the patch can be included before Buster release.

**Comment by neuffer** [comment]
Please let me know when you open the bug.

**Comment by killerbees19** [comment]
Same issue with `show_vacation` and `show_disabled` from commit 9d31639:

```
Undefined index: show_vacation in /usr/share/postfixadmin/functions.inc.php on line 2141
Undefined index: show_disabled in /usr/share/postfixadmin/functions.inc.php on line 2151
```

Sorry, I didn't see it earlier…

> Thanks for finding that. I need better tests....

Maybe :grin:

**Comment by DavidGoodwin** [comment]
477ede0175abb100889bcc594ee95885503aa5a2

**Comment by killerbees19** [comment]
btw: For all Debian fanboys…

```
 postfixadmin (3.2.1-2) unstable; urgency=medium
 .
   * remove backported code which results in errors (closes: #920564)
```

I'm closing this issue now, thanks for fixing @DavidGoodwin! :-)

**Comment by DavidGoodwin** [comment]
Thanks @killerbees19
